src/Storage/Repository/PgInstanceRepository.cs

```diff
@@ -83,6 +85,7 @@ public async Task<Instance> Create(Instance instance, CancellationToken cancella
             pgcom.Parameters.AddWithValue("_appid", NpgsqlDbType.Text, instance.AppId);
             pgcom.Parameters.AddWithValue("_taskid", NpgsqlDbType.Text, instance.Process?.CurrentTask?.ElementId ?? (object)DBNull.Value);
             pgcom.Parameters.AddWithValue("_altinnmainversion", NpgsqlDbType.Integer, altinnMainVersion);
+            pgcom.Parameters.AddWithValue("_confirmed", NpgsqlDbType.Boolean, instance.CompleteConfirmations != null && instance.CompleteConfirmations.Any(c => c.StakeholderId == instance.Org));
```

Comment:
With this the _confirm parameter is never null. It's always either true or false. Could that simplify the UPDATE statement?